python/utilities/ev_whitelist/generate_ev_whitelist.py

```diff
@@ -66,7 +68,8 @@ def _write_cert_and_chain(
              "wb"))
 
 def _ev_match(
-        output_dir, certificate, entry_type, extra_data, certificate_index):
+        output_dir, last_entry_index, certificate, entry_type, extra_data,
+        certificate_index):
     """Matcher function for the scanner. Returns the certificate's hash if
     it is a valid EV certificate, None otherwise."""
```

Comment:
Comment needs revising as it's now possible for a cert to be a valid EV cert yet not get returned because of its index being deemed 'not acceptable'